### ISSUE TITLE
关于位置编码章节的疑问

### ISSUE THREAD

**Issue by LeeQueue513** [Tongyun1/from-minimind-to-more#9]
作者你好！在阅读位置编码章节时我有几个疑问想跟您请教一下：

1. 在【5.3 线性内插的代价：分辨率危机】部分，笔记提到：

> 对于高频分量（转得快的维度），这可能影响不大。但对于低频分量，甚至是中频分量，这种压缩会导致相邻Token在向量空间中“靠得太近”，难以区分。 这就好比你把一张高分辨率图片缩小了，虽然内容还在，但像素变得模糊了。模型在处理精细的局部关系（如相邻词的语法依赖）时，会变得迟钝。这解释了为什么线性内插在扩展倍数很大时，短文本的性能会下降 。

这里我有点不理解，根据后续的NTK和YaRN来看，高频应该是尽量不要压缩的，而低频需要被压缩（防止超过训练范围）。笔记的6.1也提到低频稍微压缩一下不会丢失太多信息。所以，上面贴的笔记5.3里是不是说反了？

2. 在【7.2 熵理论与温度缩放】部分，笔记提到：

> 当 $s$ 很大时，分母变大，相当于降低了Attention Logits的数值，增加了Softmax的熵（使分布更平滑），强迫模型分散注意力，从而覆盖更长的上下文。

这里由于公式是 $\text{Attention}(Q, K) = \text{Softmax}\left(\frac{QK^T}{t\sqrt{d_k}}\right)$ 和 $\sqrt{\frac{1}{t}} = 0.1 \ln(s) + 1$，我理解的是当 s 变大的时候 t 会变小，那么softmax应该更尖锐了才对呀，为什么是更平缓了呢？
